utils/lite-components-testkit/src/main/scala/pl/touk/nussknacker/engine/lite/util/test/LiteKafkaTestScenarioRunner.scala

```diff
@@ -0,0 +1,93 @@
+package pl.touk.nussknacker.engine.lite.util.test
+
+import com.typesafe.config.ConfigValueFactory.fromAnyRef
+import com.typesafe.config.{Config, ConfigFactory, ConfigValueFactory}
+import io.confluent.kafka.schemaregistry.ParsedSchema
+import io.confluent.kafka.schemaregistry.avro.AvroSchema
+import io.confluent.kafka.schemaregistry.client.SchemaRegistryClient
+import org.apache.avro.Schema
+import org.apache.avro.generic.GenericRecord
+import org.apache.kafka.clients.consumer.ConsumerRecord
+import org.apache.kafka.clients.producer.ProducerRecord
+import pl.touk.nussknacker.engine.api.component.ComponentDefinition
+import pl.touk.nussknacker.engine.avro.AvroUtils
+import pl.touk.nussknacker.engine.avro.schemaregistry.confluent.ConfluentUtils
+import pl.touk.nussknacker.engine.graph.EspProcess
+import pl.touk.nussknacker.engine.util.test.TestScenarioRunner
+
+import scala.reflect.ClassTag
+
+object LiteKafkaTestScenarioRunner {
+  val DefaultKafkaConfig: Config =
+    ConfigFactory
+      .empty()
+      .withValue("kafka.kafkaAddress", ConfigValueFactory.fromAnyRef("kafka:666"))
+      .withValue("kafka.kafkaProperties.\"schema.registry.url\"", fromAnyRef("schema-registry:666"))
+
+  def apply(schemaRegistryClient: SchemaRegistryClient, components: List[ComponentDefinition]): LiteKafkaTestScenarioRunner =
+    new LiteKafkaTestScenarioRunner(schemaRegistryClient, components, DefaultKafkaConfig)
+}
+
+class LiteKafkaTestScenarioRunner(schemaRegistryClient: SchemaRegistryClient, components: List[ComponentDefinition], config: Config) extends TestScenarioRunner {
```

Comment:
Why you extends `TestScenarioRunner`? Because of that you need to have Input and Output implemented. What it gives to you?

Reply:
So maybe question is: do we really need `TestScenarioRunner`?

Reply:
IMO as long as it gave something to us it was good to have it. One advantage that I saw was common method of interacting with test mechanism. But when it started to be problematic and disadvantages (unnecessary complexity) exceeded advantages, we should decide to get rid of this.

Reply:
One of main purposes of TestScenarioRunner is that runners are easily interchangeable so that you could run test either on lite or flink without deep test refactoring, thus I'm not 100% sure should we hide schemaRegistryClient inside LiteKafkaTestScenarioRunner

Reply:
@dswiecki after longer conversation with @mproch and @lciolecki we've decided to keep this `TestScenarioRunner` trait as a "marking" trait for all test runners and have a little bit different methods in kafka implemenation. We will try to keep all implementations in common convention but with slight differences. Do you agree with that approach?